src/Api/Suppression.php

```diff
@@ -41,8 +42,13 @@ class Suppression
      */
     private $hydrator;
 
-    public function __construct(ClientInterface $httpClient, RequestBuilder $requestBuilder, Hydrator $hydrator)
+    public function __construct($httpClient, RequestBuilder $requestBuilder, Hydrator $hydrator)
```

Comment:
Lets add a doc block that describes `$httpClient`. Same on `Webhook`

Reply:
The doc block was added